### PR TITLE
CLOUD-78132 do not connect Hive to LDAP

### PR DESCRIPTION
**Pull request by akanto**

### CODE REVIEW

File: core/src/main/resources/defaults/blueprints/hdp26-edw-analytics-shared.bp [hunk 1]
```diff
@@ -104,10 +104,6 @@
           "hive.merge.mapfiles": "true",
           "hive.server2.tez.initialize.default.sessions": "true",
           "hive.server2.transport.mode": "http",
-          "hive.server2.authentication.ldap.url":"{{ LDAP_URL }}",
-          "hive.server2.authentication.ldap.Domain":"{{ LDAP_DOMAIN }}",
-          "hive.server2.authentication.ldap.baseDN": "{{ LDAP_SYNC_SEARCH_BASE }}",
-          "hive.server2.authentication":"LDAP",
```
Comment (shwethags) [inline core/src/main/resources/defaults/blueprints/hdp26-edw-analytics-shared.bp]: Need to remove the corresponding configs from hive-interactive-site as well:
https://github.com/sequenceiq/cloudbreak/blob/master/core/src/main/resources/defaults/blueprints/hdp26-edw-analytics-shared.bp#L72

**Comment by shwethags** [comment]
Need to update hdp26-etl-edw-spark2-shared.bp as well

File: core/src/main/resources/defaults/blueprints/hdp26-edw-analytics-shared.bp (hunk 1 above)
Comment (akanto) [inline core/src/main/resources/defaults/blueprints/hdp26-edw-analytics-shared.bp]: Thanks. Deleted.

File: core/src/main/resources/defaults/blueprints/hdp26-shared-services.bp [hunk 2]
```diff
@@ -22,11 +22,11 @@
     },
     {
       "name": "LDAP_URL",
-      "referenceConfiguration": "hive.server2.authentication.ldap.url"
+      "referenceConfiguration": "ranger.usersync.ldap.url"
```
Comment (shwethags) [inline core/src/main/resources/defaults/blueprints/hdp26-shared-services.bp]: Probably need to change these in ephemeral cluster blueprints as well?

Comment (akanto) [inline core/src/main/resources/defaults/blueprints/hdp26-shared-services.bp]: No they needs to be deleted, they do not make sense at all.  I have raised this issue: https://hortonworks.jira.com/browse/BUG-78138

**Pull request merged**